db/migrate/20230919194857_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] It would be nice if you could make the type of the `bio` field be of type `text` as it will be taking a lengthy text about a user and the `text` type  is best for that instead of the `string` type. Kindly use `t.text :bio` 👍🏽